src/main/resources/lang/en.xml

```diff
@@ -366,4 +362,19 @@
         <snowflake>snowflake</snowflake>
         <snowflakes>snowflakes</snowflakes>
     </snowflakes>
+    <death>
```

Comment:
The reason is ocn's xml will specify the death message translations with that paths, i can remove the click messages and that, but the waiting messages should stay with that path